src/positions/types.ts

```diff
@@ -11,13 +11,32 @@ export interface PositionDisplayProps {
 
 type DataProps = EarnDataProps
 
+interface YieldRate {
+  percentage: number
+  label: string
+  tokenId: string
+}
+
+interface EarningItem {
+  amount: SerializedDecimalNumber
+  label: string
+  tokenId: string
+  subtractFromDeposit?: boolean
+}
+
 interface EarnDataProps {
-  apy: number
+  contractCreatedAt?: number
```

Comment:
would be good to document what number this is. E.g., epoch seconds, or epoch milliseconds, or something else

Reply:
this hasn't been implemented in hooks yet so we can still choose what we want it to be, is there an advantage of one over the other? It seems like we don't need millisecond accuracy since we will just show how many days old the pool is, so maybe epoch seconds would be okay?